public/js/loader.js

```diff
@@ -58,6 +60,7 @@ class Loader {
       setTimeout(() => loader.remove(), 500);
```

Comment:
Should we use requestIdleCallback after the timeout? A bunch of those finishing at the same time could introduce jank.